addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -245,9 +229,9 @@
         // -------  v11 Layout + generic layouts (~v10) -------
         // Kanban Record - Dropdown
         .o_dropdown_kanban {
-            @include o-position-absolute($o-kanban-inside-vgutter, $o-kanban-inside-hgutter);
+            @include o-position-absolute(var(--Kanban__record-padding-v, #{$o-kanban-inside-vgutter}), var(--Kanban__record-padding-h, #{$o-kanban-inside-hgutter}));
```

Comment:
Readability is not very good :/
I wouldn’t  mind to use multiple lines for rules like these ones.

Something like:

```scss
@include o-position-absolute(
    var(--Kanban__record-padding-v, #{$o-kanban-inside-vgutter}),
    var(--Kanban__record-padding-h, #{$o-kanban-inside-hgutter})
);
```

edit:  like you did [here](https://github.com/odoo/odoo/pull/96195/files#diff-36fa541f40d0f6f6022a17a8a65cde362963124b98fbd8a715b95fee5d132008R201)